dotcom-rendering/src/model/enhanceLists.ts

```diff
@@ -32,6 +32,33 @@ const constructQAndAExplainer =
 		return [];
 	};
 
+const constructMiniProfile =
+	(enhanceElements: ElementsEnhancer) =>
+	({
+		title,
+		elements,
+		bio,
+		endNote,
+	}: {
+		title?: string;
+		elements: FEElement[];
+		bio?: string;
+		endNote?: string;
+	}) => {
+		// if the element is missing its title for any reason, we will skip it
```

Comment:
What about the empty string, i.e. `title.trim() === ""`?

Reply:
Good point, handled now.